### PR TITLE
Add description about `SUPER` privilege

### PR DESCRIPTION
**Pull request by CbcWestwolf**
### What is changed, added or deleted? (Required)

添加关于 `SUPER` 权限的说明。

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): close https://github.com/pingcap/docs-cn/issues/13145

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: privilege-management.md [hunk 1]
```diff
@@ -278,6 +278,11 @@ SHOW GRANTS FOR `rw_user`@`192.168.%`;
 
 若要查看全部的动态权限，请执行 `SHOW PRIVILEGES` 语句。由于用户可使用插件来添加新的权限，因此可分配的权限列表可能因用户的 TiDB 安装情况而异。
 
+## `SUPER` 权限
```
Comment (shichun-0415) [inline privilege-management.md:281]: 是否需要把 `SUPER` 权限这个章节放到 `TiDB 各操作需要的权限` 下作为一个子章节？

File: privilege-management.md [hunk 2]
```diff
@@ -278,6 +278,11 @@ SHOW GRANTS FOR `rw_user`@`192.168.%`;
...
+
+1. 拥有 `SUPER` 权限的用户能完成几乎所有的操作，默认情况下只有 `root` 用户拥有该权限。请谨慎向其它用户授予 `SUPER` 权限。
+2. `SUPER` 权限[在 MySQL 8.0 中被认为是过时的](https://dev.mysql.com/doc/refman/8.0/en/privileges-provided.html#dynamic-privileges-migration-from-super)，可以通过 [动态权限](#动态权限) 替代 `SUPER` 权限进行更细粒度的权限控制。
```
Comment (shichun-0415) [inline privilege-management.md:284]:
```suggestion
- 拥有 `SUPER` 权限的用户能完成几乎所有的操作，默认情况下只有 `root` 用户拥有该权限。请谨慎向其它用户授予 `SUPER` 权限。
- `SUPER` 权限[在 MySQL 8.0 中被认为是过时的](https://dev.mysql.com/doc/refman/8.0/en/privileges-provided.html#dynamic-privileges-migration-from-super)，可以通过[动态权限](#动态权限)替代 `SUPER` 权限进行更细粒度的权限控制。
```

File: privilege-management.md (hunk 1 above)
Comment (CbcWestwolf) [reply privilege-management.md:281]: 不用。`TiDB 各操作需要的权限` 是按操作来划分的，如果是按权限来划分就可以放进去。

**Review by shichun-0415 — APPROVED** [review]
LGTM

**Review by xhebox — APPROVED** [review]

**Pull request merged**